tests/triggers/rbac.yaml

```diff
@@ -64,3 +64,21 @@ apiVersion: v1
 kind: ServiceAccount
 metadata:
   name: tekton-triggers
+---
+apiVersion: v1
```

Comment:
I am guessing these are created in guest cluster?

Reply:
No, on host-cluster.